### ISSUE TITLE
Add to stackage

### ISSUE THREAD

**Issue by TeofilC** [haskellari/strict-containers#5]
Thanks for this package. It would be convenient for to have it on Stackage. 

Could you open an MR with Stackage to add it to the package set? 

I understand that this adds a further maintenance burden, so if you'd like I can maintain the Stackage entry and open MR's here when bounds need to be updated, etc.

**Comment by infinity0** [comment]
Why does the implementation of stackage require package maintainers to have to do non-zero work beyond what is already done for cabal? Can't it all just be automated?

**Comment by TeofilC** [comment]
Yeah there isn't any extra work per se. I think it encourages making things compatible more promptly as otherwise packages get kicked from nightly

**Comment by infinity0** [comment]
I'm happy if you add this package to Stackage and maintain it. I've never used it myself and don't have a pressing need to, so it wouldn't be great for me to do it. I see stackage as analogously external to nix packages for example, and I'm not responsible for those either.

Yes, feel free to file MRs for version bounds if stackage uncovers those issues.
